src/styles/main.scss

```diff
@@ -0,0 +1,61 @@
+@import './utils/variables';
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+  margin: 0;
+}
+
+.calendar {
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: $width-calendar;
```

Comment:
The variable `$width-calendar` is used here. Make sure it is defined in the imported `variables` file, otherwise, this will cause a compilation error.